onnxruntime/core/platform/windows/env.cc

```diff
@@ -190,13 +190,13 @@ class WindowsEnv : public Env {
         CreateFileW(file_path, FILE_READ_ATTRIBUTES, FILE_SHARE_READ, NULL, OPEN_EXISTING, FILE_ATTRIBUTE_NORMAL, NULL)};
 #endif
     if (file_handle.get() == INVALID_HANDLE_VALUE) {
-      const int err = GetLastError();
-      return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "open file ", ToMBString(file_path), " fail, errcode = ", err);
+      const auto error_code = GetLastError();
+      return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "open file ", ToMBString(file_path), " fail, errcode = ", error_code, " - ", std::system_category().message(error_code));
```

Comment:
Honestly, I don't know how "std::system_category().message()" works.  I thought we need to get a handle of "ntdll.dll" then get the error message from it.  But your solution is much simpler.